src/_fade.scss

```diff
@@ -1,46 +1,81 @@
 /*
 * @author ivy
 * @module fade
-* @description anything to do with opacity/transparency
-* @note : this is just test code. feel free to change, I made it to illustrate 
-* how the documentation and project might work
+* @description css for manipulating opacity - fade in and out
 */
 
+
+/*
+	use keyframes fade for animation on page load
+*/
 @keyframes fade--in {
-    0% { opacity: 0; }
-    100% { opacity: 1; }
+	0% { opacity: 0; }
+	100% { opacity: 1; }
 }
 
 @keyframes fade--out {
-    0% { opacity: 1; }
-    100% { opacity: 0; }
+	0% { opacity: 1; }
+	100% { opacity: 0; }
 }
 
 .anim-fade--in {
-	animation: fade--in $duration-m ease-in;
-	animation-iteration-count: 1;
+	animation-delay:2s;
+	animation:fade--in $duration--long $easing--enter;
+	animation-iteration-count:1;
+	animation-fill-mode:forwards;
+}
+
+.anim-fade--in--short {
+	animation:fade--in $duration--short $easing--enter;
+	animation-iteration-count:1;
 	animation-fill-mode:forwards;
 }
 
 .anim-fade--out {
-	animation:fade--out $duration-m ease-out;
+	animation:fade--out $duration--medium $easing--exit;
 	animation-iteration-count:1;
 	animation-fill-mode:forwards;
 }
 
-.anim-opacity--0 {
-    opacity:0;
+.anim-fade--out--short {
+	animation:fade--out $duration--short $easing--exit;
+	animation-iteration-count:1;
+	animation-fill-mode:forwards;
+}
+
+/*
+	opacity modifiers	
+*/
+.opacity--0 {
+	opacity:0;
+}
+
+.opacity--1 {
+	opacity:1;
+}
+
+/*
+	transition css 
+	used for js, in response to events and user interaction
+*/
+
+.trans-opacity--enter {
+	transition: opacity $duration--medium $easing--enter;
+}
+
+.trans-opacity--enter--short {
+	transition: opacity $duration--short $easing--enter;
 }
 
-.anim-opacity--1 {
-    opacity:1;
+.trans-opacity--exit {
+	transition: opacity $duration--medium $easing--exit;
 }
 
-// for popover example
-.anim-opacity--short {
-    transition: opacity $duration-s ease-in-out;
+.trans-opacity--exit--short {
+	transition: opacity $duration--short $easing--exit;
 }
```

Comment:
so with the introduction of specific easings - classes have multiplied.  Rather than having just
`.trans-opacity--DURATION` there are now specific rules with each easing. 

* is this ok? or do we prefer an individual class/modifier with just a timing definition - something like:
```
.trans-opacity--short {
transition: opacity $duration--short;
}
.easing--enter {
transition-timing-function: $easing--enter;
}
```